docs/design/autoware-architecture/localization/index.md

```diff
@@ -1,5 +1,294 @@
+- The twist estimator produces velocity and angular velocity from internal sensor observation
+- The accel estimator produces acceleration and angular acceleration from internal sensor observations
+
+#### Kinematics Fusion Filter
+
+- Produces the likeliest pose, velocity, acceleration, and their covariances, computed by fusing two kinds of information:
+  - The pose obtained from the pose estimator.
+  - The velocity and acceleration obtained from the twist-accel estimator
+- Produces tf of map to base_link according to the pose estimation result
+
+#### Localization Diagnostics
+
+- Monitors and guarantees the stability and reliability of pose estimation by fusing information obtained from multiple localization modules
+- Reports error status to the error monitor
+
+#### TF tree
+
+![tf-tree](../image/localization/tf-tree.png)
+
+|   frame   | meaning                                                                                        |
+| :-------: | :--------------------------------------------------------------------------------------------- |
+|   earth   | ECEF (Earth Centered Earth Fixed）                                                             |
+|    map    | Origin of the map coordinate (ex. MGRS origin)                                                 |
+|  viewer   | User-defined frame for rviz                                                                    |
+| base_link | Reference pose of the ego-vehicle (projection of the rear-axle center onto the ground surface) |
+|  sensor   | Reference pose of each sensor                                                                  |
+
+Developers can optionally add other frames such as odom or base_footprint as long as the tf structure above is maintained.
+
+### The localization module's ideal functionality
+
+- The localization module should provide pose, velocity, and acceleration for control, planning, and perception.
+- Latency and stagger should be sufficiently small or adjustable such that the estimated values can be used for control within the ODD (Operational Design Domain).
+- The localization module should produce the pose on a fixed coordinate frame.
+- Sensors should be independent of each other so that they can be easily replaced.
+- The localization module should provide a status indicating whether or not the autonomous vehicle can operate with the self-contained function or map information.
+- Tools or manuals should describe how to set proper parameters for the localization module
+- Valid calibration parameters should be provided to align different frame or pose coordinates and sensor timestamps.
+
+### KPI
+
+To maintain sufficient pose estimation performance for safe operation, the following metrics are considered:
+
+- Safety
+  - The percentage of distance which the pose estimation satisfies the required accuracy per the overall distances defined in ODD
```

Comment:
Suggested change based on the conversation above.

```suggestion
  - The distance travelled within the ODD where pose estimation met the required accuracy, divided by the overall distance travelled within the ODD, as a percentage.
```

Reply:
Resolved in e7aa1adf